### ISSUE TITLE
Will it be released?

### ISSUE THREAD

**Issue by lometsj** [grant-h/ShannonEE#1]
such a long time has gone.

**Comment by mariusmue** [comment]
Yes, we are painfully aware of this! We hope that we can release all the code in the upcoming months - Unfortunately, we need to hold back the release for the time being, but are working internally already on documentation and other usability features for the release!

**Comment by bkerler** [comment]
Any updates on the release ? :D

**Comment by mariusmue** [comment]
Heya, thanks for checking back! Indeed, we are wrapping up and are currently writing documentation. The release should happen over here: https://github.com/FirmWire/FirmWire, and we hope to get the things public by the end of this week!

**Comment by mariusmue** [comment]
Aaaaand we did release it! Check over here: https://github.com/FirmWire/FirmWire.

I'll open these days a PR to reflect it in the Readme.md of this repository.
